provider/hybrik/hybrik.go

```diff
@@ -0,0 +1,500 @@
+			manifestOutputDir = path.Join(manifestOutputDir, manifestSubDir)
+		}
+
+		manifestElement := hwrapper.Element{
+			UID:  "manifest_creator",
+			Kind: "manifest_creator",
+			Payload: hwrapper.ManifestCreatorPayload{
+				Location: hwrapper.TranscodeLocation{
+					StorageProvider: "s3",
+					Path:            manifestOutputDir,
+				},
+				FilePattern: manifestFilePattern,
+				Kind:        hls,
+			},
+		}
+
+		cj.Payload.Elements = append(cj.Payload.Elements, manifestElement)
+
+		var manifestFromConnections []hwrapper.ConnectionFrom
+		for _, hlsElementID := range hlsElementIds {
+			manifestFromConnections = append(manifestFromConnections, hwrapper.ConnectionFrom{Element: "transcode_task" + strconv.Itoa(hlsElementID)})
+		}
+
+		cj.Payload.Connections = append(cj.Payload.Connections,
+			hwrapper.Connection{
+				From: manifestFromConnections,
+				To: hwrapper.ConnectionTo{
+					Success: []hwrapper.ToSuccess{
+						{Element: "manifest_creator"},
+					},
+				},
+			},
+		)
+
+	}
+
+	resp, err := json.Marshal(cj)
+	if err != nil {
+		return "", err
+	}
+
+	return string(resp), nil
+}
+
+func (hp *hybrikProvider) JobStatus(job *db.Job) (*provider.JobStatus, error) {
+
+	ji, err := hp.c.GetJobInfo(job.ProviderJobID)
+	if err != nil {
+		return &provider.JobStatus{}, err
+	}
+
+	var status provider.Status
+	switch ji.Status {
+	case active:
+		fallthrough
+	case activeRunning:
+		fallthrough
+	case activeWaiting:
+		status = provider.StatusStarted
+	case queued:
+		status = provider.StatusQueued
+	case completed:
+		status = provider.StatusFinished
+	case failed:
+		status = provider.StatusFailed
+	}
+
+	return &provider.JobStatus{
+		ProviderJobID: job.ProviderJobID,
+		ProviderName:  hp.String(),
+		Progress:      float64(ji.Progress),
+		Status:        status,
+	}, nil
+}
+
+func (hp *hybrikProvider) CancelJob(id string) error {
+	return hp.c.StopJob(id)
+}
+
+func (hp *hybrikProvider) CreatePreset(preset db.Preset) (string, error) {
+	var minGOPFrames, maxGOPFrames, gopSize int
+
+	gopSize, err := strconv.Atoi(preset.Video.GopSize)
+	if err != nil {
+		return "", err
+	}
+
+	if preset.Video.GopMode == "fixed" {
+		minGOPFrames = gopSize
+		maxGOPFrames = gopSize
+	} else {
+		maxGOPFrames = gopSize
+	}
+
+	container := ""
+	for _, c := range hp.Capabilities().OutputFormats {
+		if preset.Container == c || (preset.Container == "m3u8" && c == hls) {
+			container = c
+		}
+	}
+
+	if container == "" {
+		return "", ErrUnsupportedContainer
+	}
+
+	bitrate, err := strconv.Atoi(preset.Video.Bitrate)
+	if err != nil {
+		return "", ErrBitrateNan
+	}
+
+	audioBitrate, err := strconv.Atoi(preset.Audio.Bitrate)
+	if err != nil {
+		return "", ErrBitrateNan
+	}
+
+	var videoWidth *int
+	var videoHeight *int
+
+	if preset.Video.Width != "" {
+		var presetWidth int
+		presetWidth, err = strconv.Atoi(preset.Video.Width)
+		if err != nil {
+			return "", ErrVideoWidthNan
+		}
+		videoWidth = &presetWidth
+	} else {
+		videoWidth = nil
```

Comment:
Same as before: ``videoWidth`` is already nil, so you can remove this ``else`` block.